RELEASE.md

```diff
@@ -1,6 +1,12 @@
 # Upcoming release:
 Add release notes for the upcoming release here.
 
+## Major features and improvements
+
+- renamed `get_data_from_kedro` to `format_pipeline_data` (#72)
+- Added pipeline, and catalog arguments to `format_pipeline_data` (#72)
```

Comment:
@tolomea I think Waylon has already added release notes? It's not super critical anyway - provided the PR description is decent I usually add it later on during the release process